src/main/resources/assets/indrev/lang/zh_cn.json

```diff
@@ -180,255 +180,255 @@
   "item.indrev.wrench.input": "输入",
   "item.indrev.wrench.input_output": "输入和输出",
   "item.indrev.wrench.none": "无",
-  "item.indrev.wrench.tooltip": "Shift+右击 更改模式",
-  "item.indrev.wrench.mode": "模式：%s",	   
+  "item.indrev.wrench.tooltip": "Shift+右击以更改模式",
+  "item.indrev.wrench.mode": "模式：%s",
   "item.indrev.tech_soup": "科技汤",
   "item.indrev.scan_output": "扫描结果",
   "item.indrev.portable_charger": "便携充电器",
-  "item.indrev.gamer_axe": "游戏爱好者之斧",  
+  "item.indrev.gamer_axe": "头号玩家斧",
   "gui.widget.energy": "能量",
   "gui.widget.temperature": "温度",
-  "gui.widget.temperature_info.high": "过高",
-  "gui.widget.temperature_info.medium": "适中",
-  "gui.widget.temperature_info.low": "温度不足",
+  "gui.widget.temperature_info.high": "过热",
+  "gui.widget.temperature_info.medium": "最适",
+  "gui.widget.temperature_info.low": "过冷",
   "gui.indrev.heatgen.idle": "闲置中……",
   "gui.indrev.heatgen.title": "发电中……",
-  "gui.indrev.heatgen.lftick": "%s LF/tick",
-  "gui.indrev.heatgen.extra": "请注意，温度过高！",
+  "gui.indrev.heatgen.lftick": "%s LF/刻",
+  "gui.indrev.heatgen.extra": "小心，烫！",
   "gui.indrev.guide_book_shortcut.contains": "打开工业革命手册",
   "gui.indrev.guide_book_shortcut.missing": "缺少工业革命手册！",
   "gui.indrev.tooltip.maxInput": "最大输入：",
   "gui.indrev.tooltip.maxOutput": "最大输出：",
   "gui.indrev.tooltip.maxEnergyStored": "最大储能：",
   "gui.indrev.tooltip.energyCost": "能耗：",
-  "gui.indrev.tooltip.processSpeed": "速度：",
+  "gui.indrev.tooltip.processSpeed": "速率：",
   "gui.indrev.tooltip.ratio": "发电：",
   "gui.indrev.tooltip.press_shift": "按 Shift 查看详细信息",
-  "gui.indrev.tooltip.lftick": "%s LF/tick",
+  "gui.indrev.tooltip.lf刻": "%s LF/刻",
```

Comment:
this is concarn plz fix

Reply:
fixx

Reply:
https://github.com/GabrielOlvH/Industrial-Revolution/pull/58/commits/807be087c82c3ea7ccefb26d3931e8bb27e7416a
here, github thank :microthonk:

Reply:
~~why is it not on the pr lolz~~